lib/gitter-adapter.js

```diff
@@ -109,6 +109,13 @@ Adapter.prototype.setup = function(token) {
   this.loggableToken = obfuscateToken(token);
   debug('setup', { token: this.loggableToken });
 
+  if (token === "dbff95de400bdb378e908dc6ad7b51b029bb493d") {
+      var mask = ':gitter!gitter@irc.gitter.im';
```

Comment:
should we log it out so we can see it happening and confirm it's working? we could also log out the user name (can't figure out where to find it) and perhaps we can contact the user and say, OY, stop being a naughty boy.